code/modules/projectiles/guns/energy/beam_rifle.dm

```diff
@@ -61,6 +60,7 @@
 	var/projectile_setting_pierce = TRUE
 	var/delay = 25
 	var/lastfire = 0
+	///The last mouse parameters, stored
```

Comment:
do autodocs work if its below the var?

Reply:
Oh, that's a leftover from something I've had in mind but scratched. I'll delete it.